From ce25cea7e8d10f9c318e2e7ef1dc1013921ed062 Mon Sep 17 00:00:00 2001
From: Simon Conseil <contact@saimon.org>
Date: Thu, 5 Dec 2013 23:42:27 +0100
Subject: [PATCH] Use print() function to fix install on python 3

clint 0.3.2 can't be installed on python 3.3 because of a print statement.
---
 clint/textui/prompt.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/clint/textui/prompt.py b/clint/textui/prompt.py
index 7c4bbdd..64d60bf 100644
--- a/clint/textui/prompt.py
+++ b/clint/textui/prompt.py
@@ -8,7 +8,7 @@
 
 """
 
-from __future__ import absolute_import
+from __future__ import absolute_import, print_function
 
 from re import match, I
 
@@ -30,7 +30,7 @@ def yn(prompt, default='y', batch=False):
         if not batch:
             input = raw_input(prompt).strip()
         else:
-            print prompt
+            print(prompt)
             input = ''
 
         # If input is empty default choice is assumed
-- 
1.8.5.1

